### DISCUSSION TITLE
Dax Raad - SST

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#206] [Announcements]
- https://twitter.com/thdxr
- https://twitter.com/SST_dev
- https://twitter.com/tomorrow_fm
- https://github.com/sst/sst
- https://thdxr.com
- https://seed.run/

#### SST

- [x] Why do you think Amazon makes their services so hard to use?
- [x] What is SST?
- [x] What are the benefits to using AWS in this way?
- [x] Why not use CDK directly?
- [x] What are some of the primitives it provides? How does this compare to doing it manually?
- [x] How does it integrate with front end frameworks?
- [x] Why would I choose using this + nextjs over just using vercel?
- [x] Do I still need to read the AWS docs or does SST distill their docs into something more consumable?
- [x] Does it support all the AWS services? What does it look like when I go off the preconfigured components?
- [x] What's a stack?
- [x] What was the most challenging component to get right?
- [x] What does local development look like?
- [x] Does it support multiple env?
- [x] What testing tools are provided?

#### SST Console 

- [x] What is SST Console (previously seed)?
- [x] What are some standout features?

### Future

- [x] You have a lot of them, but what's your spiciest dev take?
- [x] What do you think the future of backend is? A move to the edge? All server less?

**Comment by thdxr** [comment]
We're in the process of rotating out SEED as a product and replacing it with a new one called the SST Console - can keep all the questions just wanted to point that out

**Comment by thdxr** [comment]
Tooltips:
Heat It: https://www.amazon.com/dp/B0B5R3VQJ3?psc=1&ref=ppx_yo2ov_dt_b_product_details

MEATER: https://www.amazon.com/MEATER-Thermometer-Rotisserie-Bluetooth-Connectivity/dp/B07H8WTFHW/ref=sr_1_5_pp?keywords=meatter&qid=1693836888&sr=8-5&ufe=app_do%3Aamzn1.fos.006c50ae-5d4c-4777-9bc0-4513d670b6bc

PopSQL: https://popsql.com/

Replicache: https://replicache.dev/

**Comment by hipstersmoothie** [comment]
https://digest.browsertech.com/archive/browsertech-digest-how-rayon-is-making-cad/
https://github.com/infinitered/flame

**Comment by just-be-dev** [comment]
- https://github.com/memgraph/odin
- https://getdarkmatter.dev/
